src/lib/components/Collapsible.svelte

```diff
@@ -129,6 +130,12 @@
       align-items: center;
       justify-content: flex-start;
     }
+
+    outline: none;
+
+    &:focus {
+      filter: contrast(1.25);
```

Comment:
Out of curiosity, is it noticeable on text?

Reply:
Not really but, did not had good idea. `text-decoration: underline;` or better idea?

Reply:
Thanks, I think it's good as it is 👍

Maybe something for later to make tab-navigation focus more prominent e.g.
```
&:focus-visible {
  outline: var(--padding-0_25x) solid var(--warning-emphasis);
  outline-offset: calc(-1*var(--padding-0_25x));
}
```
<img width="503" alt="image" src="https://user-images.githubusercontent.com/98811342/236386464-77a6ca7a-db51-46bf-a8e4-97cf66ea5c00.png">

Reply:
That's a great idea! If you want to give it a try later, go for it